### PR TITLE
Add polish lang to xml files

### PR DESCRIPTION
**Pull request by pauldim**
I found couple of mistakes, and I changed them.
Are the variables case sensitive, because I found such variables in the files of one user. For example, in the description he gave %Variable1% and then in the variables he gave %variable1%.

### CODE REVIEW

**Comment by pauldim** [comment]
Haben Sie einen allgemeinen Index von Filmen, Serien, Nachrichten, Drehbüchern, um zu wissen, was bereits hinzugefügt wurde?
Wenn ich etwas Eigenes hinzufügen möchte, steht es unter dem Benutzernamen

**Comment by nittka** [comment]
Variable name casing: using variable names consistently is the safest. It may be that variable resolution is case insensitive, but I would not count on it.

The database files *are* the index of what exists. For programmes, the original title and imdb_id in the meta data may help to check if a movie already exists. We do not have any TVTower-specific tooling.
Start adding new content in a user-file. There is documentation in German for the format (entry point is https://github.com/TVTower/Documentation/blob/master/database_de/main.md). The general approach is that you look for a movie/news etc. that is close to what you want, copy it and change what needs to be changed. As always, start small (with few entries) - making changes to few entries is easier than refactoring thousands of lines.

When creating a PR for new content, it is well possible that scr0llbaer9000 will spot duplicates, as he went through the entire database recently.

File: res/database/Default/user/cujo.xml
Comment (nittka) [inline res/database/Default/user/cujo.xml:1]: There seem to be changes in this file that prevent GitHub from recognizing the additions only. I'll have to check this file separately.

**Review by nittka — COMMENTED** [review]
Checking the database files will take some more time.

File: res/database/Default/user/cujo.xml
Comment (pauldim) [reply res/database/Default/user/cujo.xml:1]: Are you going to add the user files then eventually to the files in the main default directory?

Comment (nittka) [reply res/database/Default/user/cujo.xml:1]: The user files are part of a release, so the content is available anyway. There is no established process of moving user entries to the "official" files.

**Comment by scr0llbaer9000** [comment]
> Haben Sie einen allgemeinen Index von Filmen, Serien, Nachrichten, Drehbüchern, um zu wissen, was bereits hinzugefügt wurde?

"Real" films, TV series and programs usually all end up in the database_programmes.xml file to make it easier for end users to do the search-and-replace of XML tags to play with real titles.

But when in doubt, just do a global search. You can do this with modern editors like VSCode / VSCodium.

There's no separation of fictional from real news yet. Maybe we should re-structure that a bit.

Screenplays based on "real" media (books, video games, etc) are a relatively recent addition and so far are located in the users directory in the files that end with _adaptations.xml

I have a big bunch of new content coming up. Don't create new content for now to avoid possible duplicates.

I'll do the PR when these Polish translations are done (with them merged already).

**Comment by pauldim** [comment]
Ok thanks for the information.

**Comment by nittka** [comment]
I suggest, you use this PR only for the database files from the user directory - in particular, please create a separate PR for the programme file as that is massive.
The other PR at one point contained changes to scr0llbar_news.xml which is not yet contained here, in case you still want to include it.

File: res/database/Default/user/kasi.xml [hunk 1]
```diff
@@ -1689,145 +1938,172 @@
...
 				<scripttemplate licence_type="2" product="3" index="1" guid="Kasi-script-UnterhaltungNachts-Ep2">
 					<title>
 						<de>Heute ist eine besondere Nacht</de>
 						<en>Today is a special night</en>
+						<pl>Dziś jest wyjątkowa noc</pl>
 					</title>
 					<description>
 						<de>Die Einschaltquoten des letzten Tages waren grandios. „Die Nacht%name%“ würdigt dies mit besondere Szenen.</de>
 						<en>The viewing rates of the last day were terrific. "The night %name%" honors this with special scenes.</en>
+						<pl>%name%". Pogadamy jak nie przespać nocy.</pl>
```
Comment (nittka) [inline res/database/Default/user/kasi.xml:1970]: missing opening quote

File: res/database/Default/user/kieferer.xml [hunk 2]
```diff
@@ -120,20 +133,25 @@ But the tour operator tricks them.</en>
...
 			</description>
 			<variables>
 				<modifier>
 					<de>| in %RANDOMCITY%</de>
 					<en>| in %RANDOMCITY%</en>
+					<pl>| w %RANDOMCITY%</pl>
 				</modifier>
 				<randomcity>
 					<de>%STATIONMAP:RANDOMCITY%|Prosperous Creek</de>
+					<pl>Prosperous Creek|Sakramento|Ontario|Zimnym potoku|Bloody Village</pl>
```
Comment (nittka) [inline res/database/Default/user/kieferer.xml:154]: number of alternatives must be the same

File: res/database/Default/user/kieferer.xml [hunk 3]
```diff
@@ -165,124 +183,150 @@ Any means will do.</en>
...
 					<en>Reason for this are merely satisfactory figures of %VALUES%.|Negative valuations of the %SECTION% provide for falling prices.|The economic data unsettle the stockbrokers.|On a large scale, falling prices. Only %VALUES% can convince today.|Analysts' assessments of the %SECTION% cause some disillusionment.</en>
+					<pl>Powodem tego są jedynie zadowalające dane %VALUES%.|Negatywne wyceny w %SECTION% powodują spadki cen.|Dane ekonomiczne niepokoją maklerów.|Na dużą skalę spadków cen tylko walory %VALUES% mogą dziś przekonać inwestorów.|Oceny analityków w %SECTION% powodują lekkie rozczarowanie.</pl>
 				</stockexchange3A>
 
 
 				<resources1>
 					<de>Rohstoffe heute erfreulich. %RESOURCES1A%|Rohstoffwerte wissen zu gefallen. %RESOURCES1A%</de>
 					<en>Commodities are pleasing today. %RESOURCES1A%|Commodity stocks look appealing. %RESOURCES1A%</en>
+					<pl>Rynek surowców jest dziś atrakcyjny. %RESOURCES1A%</pl>
```
Comment (nittka) [inline res/database/Default/user/kieferer.xml:256]: number of alternatives must be the same

File: res/database/Default/user/ronny.xml [hunk 4]
```diff
@@ -928,21 +1038,25 @@ Illustrious guests and music by [3|Nick].</en>
 			<title>
 				<de>${cntry} von oben</de>
 				<en>Bird's eye view of ${cntry}</en>
+				<pl>Widok z lotu ptaka na ${cntry}</pl>
 			</title>
 			<description>
 				<de>Erleben Sie atemberaubende Luftaufnahmen von ${cntry}.|${cntry} wie Sie es noch nie gesehen haben.|${cntry} aus einer neuen Perspektive.</de>
 				<en>Experience breathtaking aerial views of ${cntry}.|${cntry} as you have never seen it before.|${cntry} from a new perspective.</en>
+				<pl>Doświadcz zapierających dech w piersiach widoków z lotu ptaka na ${cntry}, których jeszcze nigdy wcześniej nie widziałeś. Z nowej perspektywy.</pl>
```
Comment (nittka) [inline res/database/Default/user/ronny.xml:1046]: number of alternatives must be the same

File: res/database/Default/user/ronny.xml [hunk 5]
```diff
@@ -928,21 +1038,25 @@ Illustrious guests and music by [3|Nick].</en>
...
 					<title>
 						<de>${cntry}s ${subject}</de>
 						<en>${subject} of ${cntry}</en>
+						<pl>${subject} z lotu ptaka</pl>
 					</title>
 					<variables>
 						<subject>
 							<de>Berge|Flüsse|Städte|Nationalparks|Wälder|Seen|Bahnhöfe|Schlösser|Parks|Inseln|Kirchen|Stadien|Ruinen|Gletscher|Wochenmärkte|Bahnstrecken</de>
 							<en>Mountains|Rivers|Cities|National Parks|Forests|Lakes|Railroad Stations|Castles|Parks|Islands|Churches|Stadiums|Ruins|Glaciers|Farmers' Markets|Railways</en>
+							<pl>Góry|Rzeki|Miasta|Parki narodowe|Lasy|Jeziora|Stacje kolejowe|Zamki|Parki|Kościoły|Stadiony|Ruiny zamków</pl>
```
Comment (nittka) [inline res/database/Default/user/ronny.xml:1059]: number of alternatives must be the same

File: res/database/Default/user/ronny.xml [hunk 6]
```diff
@@ -1147,34 +1283,41 @@ Illustrious guests and music by [3|Nick].</en>
...
 					<description>
 						<de>${dsc}</de>
 						<en>${dsc}</en>
+						<pl>Podróże przez świat. Poznajemy różne kraje. ${dsc}</pl>
 					</description>
 					<variables>
 						<epttl>
 							<de>${subject} ${cntry}s|${adjective}${subject} ${cntry}s</de>
 							<en>${subject} of ${cntry}|${adjective}${subject} of ${cntry}</en>
+							<pl>${cntry}</pl>
```
Comment (nittka) [inline res/database/Default/user/ronny.xml:1309]: number of alternatives must be the same

File: res/database/Default/user/ronny.xml [hunk 7]
```diff
@@ -1147,34 +1283,41 @@ Illustrious guests and music by [3|Nick].</en>
...
 						</epttl>
 						<!--without an empty description, the series description is used again-->
 						<dsc>
 							<de> |${cntry}s ${subject} bergen so manches Geheimnis.|Fakten und Meinungen zu ${cntry}s ${subject}.|Was macht ${cntry}s ${subject} so besonders?</de>
 							<en> |${subject} in ${cntry} carry quite a few secrets.|${subject} of ${cntry} - facts and opinions.|${subject} in ${cntry}, what makes them so special?</en>
+							<pl>${cntry} to kraj który ma bardzo dużo niezwykłych tajemnic i ciekawostek.</pl>
```
Comment (nittka) [inline res/database/Default/user/ronny.xml:1315]: number of alternatives must be the same

File: res/database/Default/user/scr0llbaer.xml [hunk 8]
```diff
@@ -516,23 +582,28 @@ A must for every housewife, a curse for hard men.</en>
...
 			<description>
 				<de>Und noch ein Showbiz-Format. Die Welt da draußen scheint nicht genug davon bekommen zu können.</de>
 				<en>And another showbiz format. The world out there can't seem to get enough of it.</en>
+				<pl>I kolejny format showbiznesowy. Świat nie może się tym nacieszyć.</pl>
 			</description>
 			<variables>
 				<pulse>
 					<de>Puls|Sprengstoff|Blitze|Buzz|Feuerwerk|Blitzlicht|Inferno|Zündstoff|Bumms|Furore</de>
 					<en>Pulse|Explosive|Flash|Buzz|Fireworks|Spotlight|Inferno|Fuse|Boom|Furore</en>
+					<pl>Impuls|Wybuch|Błysk|Fajerwerk|Flash|Inferno|Lont|Boom|Furora</pl>
```
Comment (nittka) [inline res/database/Default/user/scr0llbaer.xml:596]: number of alternatives must be the same

File: res/database/Default/user/kasi.xml [hunk 9]
```diff
@@ -195,23 +222,28 @@
...
+				<pl>Uwolnić %Tier% %Name%</pl>
 			</title>
 			<description>
 				<de>Ein Kind verliebt sich beim Zoobesuch in den %Tier% %title%%Name%. Doch sieht dieser sehr traurig aus. So tut das Kind alles in seiner Macht stehende, um ihn zu befreien.</de>
 				<en>A child falls in love with the %Tier% %title%%Name% while visiting the zoo. But it looks very sad. So the child does everything in its power to free the animal.</en>
+				<pl>Dziecko zakochuje się w zwierzątku podczas wizyty w zoo. Wygląda ono jednak na bardzo smutne. Dziecko robi więc wszystko, co w jego mocy, by uwolnić %Name%.</pl>
 			</description>
 			<variables>
-				<tier>
+				<Tier><!--tier changed to Tier.  -->
```
Comment (nittka) [inline res/database/Default/user/kasi.xml:233]: OK. You can remove the comment.

File: res/database/Default/user/kasi.xml [hunk 10]
```diff
@@ -195,23 +222,28 @@
...
 					<de>Affen|Nacktmull|Goldfisch|Löwen|Tiger|Hasen|Panda</de>
 					<en>monkey|naked mole rat|goldfish|lion|tiger|rabbit|panda</en>
-				</tier>
-				<name>
+					<pl>małpę|surykatkę|delfina|lwa|tygrysa|zająca|pandę</pl>
+				</Tier>
+				<Name><!--name changed to Name.  -->
```
Comment (nittka) [inline res/database/Default/user/kasi.xml:238]: OK. You can remove the comment.

File: res/database/Default/user/kasi.xml [hunk 11]
```diff
@@ -195,23 +222,28 @@
...
 					<de>Jingels|Wollo|Hubert|Billy|Fluffy|Bobo|Flocke</de>
 					<en>Jingels|Wollo|Hubert|Billy|Fluffy|Bobo|Flake</en>
-				</name>
+					<pl>Jingela|Wollo|Huberta|Billiego|Fluffiego|Bobo|Neo</pl>
+				</Name>
 				<title>
 					<de>Mister |Little |Big |Mac</de>
 					<en>Mister |Little |Big |Mac</en>
+					<!--In Polish, not required. I had to rewrite it to make it grammatically ok.  -->
```
Comment (nittka) [inline res/database/Default/user/kasi.xml:246]: May be OK, but the "title" actually modifies the name. MacFluffy vs. Mister Fluffy.

File: res/database/Default/user/kasi.xml [hunk 12]
```diff
@@ -266,72 +301,86 @@
...
 					<description>
 						<de>Die Gruppe wechselt geschlossen zu einem anderen Netzwerk. Doch der Mörder ist noch immer auf freiem Fuß und niemand ahnt, dass er dank %Name1% alles über die Gruppe weiß.</de>
 						<en>The complete group switches to another network. But the killer is still at large and no one suspects that he knows everything about the group because of %Name1%.</en>
+						<pl>Cała grupa przenosi się do innej sieci. Zabójca wciąż jest na wolności i nikt nie podejrzewa, że wie wszystko o grupie z poszukiwawczej.</pl>
 					</description>
 				</scripttemplate>
 			</children>
 			<variables>
-				<name1>
+				<Name1><!--name1 changed to Name1.  -->
```
Comment (nittka) [inline res/database/Default/user/kasi.xml:374]: Ok. You can remove the comment.

File: res/database/Default/user/kasi.xml [hunk 13]
```diff
@@ -266,72 +301,86 @@
...
-				</name1>
-				<name2>
+					<pl>Finstergramie|Twoterze|Focebeeku|Putube-ie|Snoopchat-cie|Whosappie</pl>
+				</Name1>
+				<Name2>
 					<de>Girl|Woman|Maid|Child|Daughter|Son|Boy|Man</de>
 					<en>Girl|Woman|Maid|Child|Daughter|Son|Boy|Man</en>
-				</name2>
+					<pl>Dziewczyna|Kobieta|Panna|Dziecko|Córka|Syn|Chłopiec|Facet</pl>
+				</Name2><!--name2 changed to Name2.  -->
```
Comment (nittka) [inline res/database/Default/user/kasi.xml:383]: OK. You can remove the comment

**Review by nittka — CHANGES_REQUESTED** [review]

File: res/database/Default/user/kasi.xml (hunk 1 above)
Comment (pauldim) [reply res/database/Default/user/kasi.xml:1970]: yes agree

File: res/database/Default/user/kieferer.xml (hunk 2 above)
Comment (pauldim) [reply res/database/Default/user/kieferer.xml:154]: It will not be grammatically consistent. Changed

Comment (pauldim) [reply res/database/Default/user/kieferer.xml:154]: I adapted it grammatically

File: res/database/Default/user/kieferer.xml (hunk 3 above)
Comment (pauldim) [reply res/database/Default/user/kieferer.xml:256]: Actually, an oversight

File: res/database/Default/user/ronny.xml (hunk 4 above)
Comment (pauldim) [reply res/database/Default/user/ronny.xml:1046]: Corrected

File: res/database/Default/user/ronny.xml (hunk 5 above)
Comment (pauldim) [reply res/database/Default/user/ronny.xml:1059]: Corrected. My mistake.

File: res/database/Default/user/ronny.xml (hunk 6 above)
Comment (pauldim) [reply res/database/Default/user/ronny.xml:1309]: corrected

File: res/database/Default/user/ronny.xml (hunk 7 above)
Comment (pauldim) [reply res/database/Default/user/ronny.xml:1315]: Corrected

File: res/database/Default/user/scr0llbaer.xml (hunk 8 above)
Comment (pauldim) [reply res/database/Default/user/scr0llbaer.xml:596]: Corrected, added

File: res/database/Default/user/kasi.xml (hunk 11 above)
Comment (pauldim) [reply res/database/Default/user/kasi.xml:246]: I'm not using title.

File: res/database/Default/user/kasi.xml (hunk 12 above)
Comment (pauldim) [reply res/database/Default/user/kasi.xml:374]: ok

File: res/database/Default/user/kasi.xml (hunk 13 above)
Comment (pauldim) [reply res/database/Default/user/kasi.xml:383]: ok

File: res/database/Default/user/cujo.xml [hunk 14]
```diff
@@ -1,764 +1,876 @@
...
+		</news>
+		<news id="cujo-news-superkerl-03" type="2" thread_id="cujo-news-superkerl" creator="8827" created_by="Cujo">
+			<title>
+				<de>Superkerl und Superweib haben geheiratet</de>
+				<en>Superguy and Superwoman got married</en>
+				<pl>Superguy i Superwoman wzięli ślub</pl>
+			</title>
+			<description>
+				<de>Superweib über Superkerl: „In der Hochzeitsnacht war er super.“</de>
+				<en>Superweib about Superguy: "On the wedding night he was super."</en> <!-- fault Superwoman not Superweib -->
```
Comment (nittka) [inline res/database/Default/user/cujo.xml:49]: fix Superweib here and remove the comment

**Comment by nittka** [comment]
After the last fix (Superweib), I would merge, if you do not want to make further changes to these files.

File: res/database/Default/user/cujo.xml (hunk 14 above)
Comment (pauldim) [reply res/database/Default/user/cujo.xml:49]: Done

**Review by nittka — APPROVED** [review]

**Pull request merged**